src/components/buttons.scss

```diff
@@ -0,0 +1,106 @@
+// Fonts
+
+@import 'fonts';
+
+.fontButton {
+  font-family: "Roboto", "Verdana";
```

Comment:
Intellij sier: `Font property font-family does not have generic default`.
Det kan altså være lurt å legge til en generic font i tilfellet Roboto eller Verdana ikke kan lastes.
Se eksempler her: http://www.w3schools.com/cssref/css_websafe_fonts.asp